lib/widgets/login.dart

```diff
@@ -161,30 +183,61 @@ class _EmailPasswordLoginPageState extends State<EmailPasswordLoginPage> {
   Widget build(BuildContext context) {
     assert(!PerAccountStoreWidget.debugExistsOf(context));
     return Scaffold(
-      appBar: AppBar(title: const Text('Log in')),
+      appBar: AppBar(title: const Text('Log in'),
+        bottom: _inProgress
+          ? const PreferredSize(preferredSize: Size.fromHeight(4),
+              child: LinearProgressIndicator(minHeight: 4))
```

Comment:
Interesting. How did you hit on this way of controlling the height?

Reply:
Ah, I should mention in a comment: `4` is the default, and I just restated it here so it's clear what the `PreferredSize` is matched to. (The `PreferredSize` is used because `AppBar.bottom` [has to be a `PreferredSizeWidget`](https://api.flutter.dev/flutter/material/AppBar/bottom.html).)